it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/DocumentationIntegrationSpec.scala

```diff
@@ -16,26 +16,47 @@
 
 package uk.gov.hmrc.tradergoodsprofiles.controllers
 
+import org.mockito.Mockito._
+import org.scalatestplus.mockito.MockitoSugar
 import org.scalatestplus.play.PlaySpec
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
-import play.api.{Application, inject}
 import play.api.http.Status.{NOT_FOUND, OK}
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.WSClient
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import play.api.{Application, inject}
 import uk.gov.hmrc.http.client.HttpClientV2
 import uk.gov.hmrc.http.test.HttpClientV2Support
+import uk.gov.hmrc.tradergoodsprofiles.config.AppConfig
 
-class DocumentationIntegrationSpec extends PlaySpec with GuiceOneServerPerSuite with HttpClientV2Support {
+class DocumentationIntegrationSpec
+    extends PlaySpec
+    with GuiceOneServerPerSuite
+    with HttpClientV2Support
+    with MockitoSugar {
 
-  lazy val wsClient: WSClient = app.injector.instanceOf[WSClient]
+  lazy val wsClient: WSClient  = app.injector.instanceOf[WSClient]
+  val mockAppConfig: AppConfig = mock[AppConfig]
 
   override def fakeApplication(): Application =
     GuiceApplicationBuilder()
       .configure("metrics.enabled" -> false)
+      .overrides(inject.bind[AppConfig].toInstance(mockAppConfig))
       .overrides(inject.bind[HttpClientV2].to(httpClientV2))
       .build()
 
+  private def configureApp(withdrawAdviceEnabled: Boolean): WSClient = {
```

Comment:
This can be removed as we already have the fakeApplication that do that. It will be enough just to set up the expectation. on the mockAppConfig:

when(mockAppConfig.withdrawAdviceEnabled).thenReturn(true/false).

Ping me if you want have a chat.